### PR TITLE
php: Add fileinfo package required by some libraries

### PR DESCRIPTION
**Pull request by aramalipoor**

### CODE REVIEW

File: dockerfiles/php/5/Dockerfile [hunk 1]
```diff
@@ -77,8 +77,9 @@ COPY supervisord.conf /
 COPY nginx.conf /etc/nginx/conf.d/default.conf
 COPY composer.json composer.lock ./
 
-# Run composer install.
-RUN composer install --no-scripts --no-autoloader --prefer-dist --no-dev \
+# Run composer install. Use prestissimo for parallel package downloads.
+RUN composer global require hirak/prestissimo && \
```
Comment (alikhajeh1) [inline dockerfiles/php/5/Dockerfile]: does this require any changes in the app code? I assume not, but just checking. would https://github.com/hirak/prestissimo#composer-authentication cause issues?

File: dockerfiles/php/7/Dockerfile [hunk 2]
```diff
@@ -15,11 +15,11 @@ ENV TZ=Asia/Tehran \
...
 # default but you can add more using the `ADDITIONAL_PACKAGES` variable above.
-RUN apk update && \
-    apk add tzdata curl bash ca-certificates rsync supervisor nginx \
-            php7 php7-fpm php7-common php7-openssl php7-session php7-bcmath \
-            php7-dom php7-tokenizer php7-pdo php7-json php7-phar php7-mbstring \
-            php7-iconv php7-zlib php7-ctype php7-xml ${ADDITIONAL_PACKAGES} && \
+RUN apk add --update --no-cache \
+        tzdata curl bash ca-certificates rsync supervisor nginx \
+        php7 php7-fpm php7-common php7-openssl php7-session php7-bcmath php7-curl \
+        php7-dom php7-tokenizer php7-pdo php7-json php7-phar php7-mbstring php7-fileinfo \
```
Comment (alikhajeh1) [inline dockerfiles/php/7/Dockerfile]: I assume fileinfo isn't available for php5?

File: dockerfiles/php/5/Dockerfile (hunk 1 above)
Comment (aramalipoor) [reply dockerfiles/php/5/Dockerfile]: It doesn't require any code changes and works out of the box. Authentication is just a general recommendation :ok_hand:

File: dockerfiles/php/7/Dockerfile (hunk 2 above)
Comment (aramalipoor) [reply dockerfiles/php/7/Dockerfile]: Yep, it's not available. It's weird, many apps are still on PHP 5, it's literally saying if you want Alpine use php7.

File: dockerfiles/php/5/Dockerfile [hunk 3]
```diff
@@ -15,11 +15,11 @@ ENV TZ=Asia/Tehran \
 
 # Install the required packages. Popular PHP packages have been included by
 # default but you can add more using the `ADDITIONAL_PACKAGES` variable above.
-RUN apk update && \
```
Comment (alikhajeh1) [inline dockerfiles/php/5/Dockerfile]: might be nice to add the "add/update at same time" and "--no-cache" to other Dockerfile examples?

**Pull request merged**